include/openmc/capi.h

```diff
@@ -163,13 +164,38 @@ int openmc_tally_set_nuclides(int32_t index, int n, const char** nuclides);
 int openmc_tally_set_scores(int32_t index, int n, const char** scores);
 int openmc_tally_set_type(int32_t index, const char* type);
 int openmc_tally_set_writable(int32_t index, bool writable);
+int openmc_weight_windows_get_index(int32_t id, int32_t* idx);
+int openmc_weight_windows_get_id(int32_t index, int32_t* id);
+int openmc_weight_windows_set_id(int32_t index, int32_t id);
+int openmc_set_weight_windows(
+  int ww_id, size_t n, const double* lower_bounds, const double* upper_bounds);
+int openmc_update_weight_windows_magic(int32_t tally_idx, int32_t ww_idx,
+  const char* value = "mean", double threshold = 1.0, double ratio = 5.0);
```

Comment:
Yeah, there are a few other places where default arguments occur in the CAPI. Still, best not to add to the problem. I'll remove these and add some documentation so the purpose of each argument is clear in case anyone looks through the header.